### ISSUE TITLE
Zillow: 403 Forbidden

### ISSUE THREAD

**Issue by ddxv** [ZacharyHampton/HomeHarvest#15]
Python 3.10.11
Versions tested: 0.2.13

What I tried to do:

```
properties: pd.DataFrame = scrape_property(
    site_name=["zillow", "realtor.com", "redfin"],
    location="85281",
    listing_type="for_rent" # for_sale / sold
)
```

Output:
`requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://www.zillow.com/homes/for_rent/85281_rb/`

Testing this URL in browser works OK.

**Comment by cullenwatson** [comment]
Try a proxy with proxy=, also you can use tryhomeharvest.com

**Comment by ddxv** [comment]
I see, I don't have a proxy on hand at the moment, but just curious, do you think changing the UA that homeharvest is using would help? What UA is it currently set to? I have only used the library once a couple days ago, so would be surprised if the blocking rule is based on IP only.

**Comment by cullenwatson** [comment]
The headers are located here at the bottom: 
https://github.com/ZacharyHampton/HomeHarvest/blob/master/homeharvest/core/scrapers/zillow/__init__.py

`Chromium";v="116", "Not)A;Brand";v="24", "Google Chrome";v="116`

It may be the cookie as well. When I made it, if I didn't have the cookie, the requests wouldn't work. Any insights @ZacharyHampton 
<img width="1030" alt="image" src="https://github.com/ZacharyHampton/HomeHarvest/assets/78247585/af887e51-4a97-4991-9563-561e60148f19">

**Comment by cullenwatson** [comment]
It's def the cookie. I changed it to an updated one from my browser and it's good now. Maybe time-based tho not sure if it's a long-term solution

**Comment by cullenwatson** [comment]
I believe we can fix this by fetching the cookies on an initial request & dynamically setting it to a fresh cookie every time for the backend endpionts

**Comment by ddxv** [comment]
> I believe we can fix this by fetching the cookies on an initial request & dynamically setting it to a fresh cookie every time for the backend endpionts

Ah, perfect, that is the only thing I would have suggested as well. Thanks for the link above as well to __init__.py. Will also check the PR to see where fixes were made. Cheers

**Comment by cullenwatson** [comment]
Should be working good, you can upgrade pip to get latest changes

**Comment by ddxv** [comment]
Yes, pulled newest changes and checked that last commit. I'm far from an expert here, but looks good to me. Also checked and working again. Thanks for the help!
